php/data/MapIterator.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * This file defines the concept of a data Provisioner.
+ *
+ * PHP Version 7
+ *
+ * @category   Data
+ * @package    Main
+ * @subpackage Data
+ * @author     Dave MacFarlane <david.macfarlane2@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+namespace LORIS\Data;
+
+class MapIterator extends \IteratorIterator {
+    protected $mapper;
+    protected $user;
+
+    public function __construct($rows, Mapper $mapper, \User $user) {
+        parent::__construct($rows);
+        $this->mapper = $mapper;
+        $this->user = $user;
+    }
+
+    public function current() {
+        $row = parent::current();
+        return $this->mapper->Map($this->user, $row);
```

Comment:
is the capital M a typo or am I missing something?
https://github.com/aces/Loris/pull/3123/files#diff-a1144eb4b87d009eef11756f313ad319R46

Reply:
There do not seem to be tests this is why i'm concerned that even though TravisCI passed this might still be an issue :s

Reply:
Yes, it should be lower case. (Function names in PHP aren't case sensitive, but they should still be camelCaps for consistency..)